Documentation/project-docs/dogfooding.md

```diff
@@ -236,4 +236,22 @@ need to modify your .csproj to target netcoreapp2.0 and reference the nightly bu
   </PropertyGroup>
 ```
 
-In a future update to Visual Studio, it will no longer be necessary to make this edit.
\ No newline at end of file
+In a future update to Visual Studio, it will no longer be necessary to make this edit.
+
+## Finding specific builds
+
+The URL scheme for the runtime is as follows:
+
+```
+https://dotnetcli.blob.core.windows.net/dotnet/master/Installers/$version$/dotnet-win-x64.$version$.exe
```

Comment:
I would use `$os$-$arch$` here instead of `win-x64`.  Since you are mix and matching x64 here and x86 below.